docs/RELEASE-NOTES.rst

```diff
@@ -1,18 +1,36 @@
 Release Notes for Container Ingress Services for Kubernetes & OpenShift
 =======================================================================
 
-Next Release
+2.7.1
 -------------
 Enhancements
 ````````````
 * Optimized processing of ConfigMaps with FilterTenants enabled
 * Added support for multihost VS policy rules for same path and service backend combination
+* Improved error handling with EDNS Custom resource
 * :issues:`1872` Support protocol UDP in Services of type LoadBalancer
 * :issues:`1918` ExternalDNS adds both VSs to a Wide IP pool
 * :issues:`2051` Fix AS3 Postdelay issue when error occurs
 * :issues:`2077` Fix recreating ARPs when CIS restarts
 * :issues:`2172` Fix Endpoint NodeName validation issue
-* :issues:`2184` Helm Chart ClusterRole Does Not Have Correct Permissions
+* Helm Chart Enhancements:
+    - issues:`2184` Helm Chart ClusterRole does not have correct permissions
+
+FIC Enhancements
+````````````````
+* Added support for FIC installation using Helm Charts, Refer `Documentation <https://github.com/F5Networks/f5-ipam-controller/blob/main/helm-charts/f5-ipam-controller/README.md>`_
+
+Known issues
+````````````
+* CIS does not delete the arp entries immediately from BigIP, When we remove all the endpoints for a service in cccl mode,
+* Unable to pass multiple infoblox labels to FIC helm charts
+* Deletion of EDNS resource not removing Wide IP config from BigIP intermittently
+* CIS sends the failed tenant declaration every 30 secs with filter-tenant parameter when a 422 error occurs in as3 response
+
+Upgrade notes
+``````````````
+* Moving from CIS > 2.6 with IPAM, see troubleshooting guide for IPAM issue ipams.fic.f5.com not found. Refer `Troubleshooting Guide <https://github.com/sravyap135/f5-ipam-controller/blob/faq/docs/faq/README.md#troubleshooting>`_
```

Comment:
can we highlight the issue like
* Moving from CIS > 2.6 with IPAM, see troubleshooting guide for IPAM issue -  `ipams.fic.f5.com not found`.

Reply:
just add " ` " to look like a code style.

Reply:
Done